### ISSUE TITLE
dates() not working

### ISSUE THREAD

**Issue by LukasNavickas** [nlp-compromise/compromise#343]
When I'm trying to parse the date from the example with 

`console.log(nlp('Finally, I just stopped caring. Luckily for me, it was 1980 and no one noticed.').dates().data())`

I don't get any results, just an empty array.

**Comment by spencermountain** [comment]
yikes! thanks, will take a look.

**Comment by spencermountain** [comment]
hey, got it.
will be fixed in `v8.2.0`
cheers

**Comment by LukasNavickas** [comment]
One more thing: would be cool if it could parse the dates from this website (under "Routes and travel dates") - http://travelfree.info/2017/03/direct-flights-verona-germany-just-e9-round-trip/

**Comment by LukasNavickas** [comment]
By the way, would you considering including cities under `places()` ?

**Comment by spencermountain** [comment]
yeah cities should work under places.
looked at the example. is getting fussed by the non-standard `–` character.
will check it out

**Comment by paulbruno** [comment]
@spencermountain (cc: @LukasNavickas): Btw, as I'm about to grab this awesome lib. to do date and place detection in English myself, and as a Comm. major, I'd just like to point out that the en-dash (–) is, in text by annoyingly fastidious English writers like myself, the most proper way to indicate a span or duration of time (what I often corrected when I was a copy editor for a regular publication), as well as other numerical spans/durations (distance, etc.). Just throwing it out there, hope this is in some way valuable!

**Comment by spencermountain** [comment]
yeah! thanks Paul. You're right. We should definitely support this.
I'll add it to v11's list, cheers.

**Comment by paulbruno** [comment]
@spencermountain Absolutely! Thanks *_so_* much for this piece of JS, man. Very new to even _attempting_ to be anything more than a consumer in the open-source ecosystem, but would love to rise to becoming a contributor, if you/the gang still in need of some additional help.
